x-pack/plugins/license_management/server/lib/license.ts

```diff
@@ -5,20 +5,21 @@
  * 2.0.
  */
 
+import { LicenseLicense } from '@elastic/elasticsearch/lib/api/types';
 import { IScopedClusterClient } from '@kbn/core/server';
 import { LicensingPluginStart } from '@kbn/licensing-plugin/server';
 
 interface PutLicenseArg {
   acknowledge: boolean;
   client: IScopedClusterClient;
   licensing: LicensingPluginStart;
-  license: { [key: string]: any };
+  licenses: LicenseLicense[];
 }
 
-export async function putLicense({ acknowledge, client, licensing, license }: PutLicenseArg) {
+export async function putLicense({ acknowledge, client, licensing, licenses }: PutLicenseArg) {
   try {
     const response = await client.asCurrentUser.license.post({
-      body: license,
+      licenses,
```

Comment:
@jloleysens It's almost two years ago, but you seemed to be the last person who touched this 😅 As far back as 6.8 this API required `licences: []` with an optional `license` property. So I suspect we were always using a deprecated / old format and the types were just finally updated to reflect that.

Reply:
Nice catch! I'll try testing your branch locally, will report soon!

Reply:
Tested locally, everything seems to be working just fine!